debezium-platform-conductor/src/test/java/io/debezium/platform/environment/actions/db/OracleTestResource.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.platform.environment.actions.db;
+
+import java.util.Map;
+
+import org.testcontainers.containers.OracleContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class OracleTestResource implements QuarkusTestResourceLifecycleManager {
+
+    private static final OracleContainer ORACLE = new OracleContainer(
+            DockerImageName.parse("gvenzl/oracle-xe:21-slim-faststart"))
```

Comment:
@Naros is it ok to use this image here?

Reply:
@mfvitale It should not go directly from Docker Hub